flamedisx/inference.py

```diff
@@ -176,24 +185,26 @@ def simulate(self, rate_multipliers=None, fix_truth=None, **params):
 
         ds = []
         for sname, s in self.sources.items():
-            rmname = sname + '_rate_multiplier'
-            if rmname in rate_multipliers:
-                rm = rate_multipliers[rmname]
-            else:
-                rm = self._get_rate_mult(sname, params)
-
-            # mean number of events to simulate, rate mult times mu source
-            mu = rm * self.mu_itps[sname](**self._filter_source_kwargs(params,
-                                                                       sname))
-            # Simulate this many events from source
-            n_to_sim = np.random.poisson(mu)
-            if n_to_sim == 0:
-                continue
-            d = s.simulate(n_to_sim,
-                           fix_truth=fix_truth,
-                           **params)
-            d['source'] = sname
-            ds.append(d)
+            if s.defaults:
```

Comment:
This is just to ignore ColumnSource for now because of #47 right? I'd put a TODO here to make sure we don't forget to remove this later.

Reply:
Yes, indeed!
I will do that!